lib/CONST.jsx

```diff
@@ -814,23 +814,23 @@ export const CONST = {
     SUBSCRIPTION_CHANGE_REASONS: {
         TOO_LIMITED: {
             id: 'tooLimited',
-            label: 'Functionality needs improvement or is confusing',
-            prompt: 'What features would you like to see added or improved?',
+            label: 'Functionality needs improvement',
+            prompt: 'What software are you migrating to and what led to this decision?',
         },
         TOO_EXPENSIVE: {
             id: 'tooExpensive',
             label: 'Too expensive',
-            prompt: 'What is the main reason you think Expensify is too expensive, and what do you think is a fair price?',
+            prompt: 'What software are you migrating to and what led to this decision?',
```

Comment:
Ah just to be sure, in the [description](https://github.com/Expensify/Expensify/issues/259131#issue-1564436048) we have this copy:

```suggestion
            prompt: 'What software are you migrating to and what do you think is a fair price?',
```

is that correct?

Reply:
Check out the comment that's linked in the description. I think this is correct

Reply:
https://github.com/Expensify/expensify-common/pull/493#issuecomment-1413689664